terraform/variables.tf

```diff
@@ -17,7 +23,7 @@ variable "ssh_key" {
 variable "ssh_user" {
   type        = string
   description = "The user that should be created and who has sudo power"
-  default     = "gila"
+  default     = "user"
```

Comment:
as the user "gila" come standard on all distro's it worked in the past ;) maybe remove this too.

Reply:
Either comment out the default or pick something more standard that is commonly used as a default. Could be "jack", "demo", ... but whatever you prefer guys